tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -158,6 +158,7 @@ def _save_snapshot(self):
                             "Geomview", "VRML", "Wavefront", "Open Inventor",
                             "Povray", "X3D"]
 
+            wildcard = ""
```

Comment:
Some pointers to documentation on these tools would be useful to have and use -- in fact it would be nice to have these documented as part of the traits manual.  It has been a decade (2005?) since I read any traits/pyface documentation, so pardon my ignorance but I think the documentation on suggested patterns for this is scarce.  Thanks!